lib/ui/pages/news_list/primary_news_list_item.dart

```diff
@@ -0,0 +1,116 @@
+import 'package:allthenews/ui/pages/news_list/primary_news_list_entity.dart';
+import 'package:flutter/material.dart';
+
+abstract class _Constants {
+  static const double newsImageRadius = 15;
+  static const double newsImageLayerOpacity = 0.7;
+  static const double authorImageRadius = 15;
+  static const double authorRightPadding = 8;
+  static const double stackItemPadding = 20;
+}
+
+class PrimaryNewsListItem extends StatelessWidget {
+  final PrimaryNewsListEntity news;
+
+  PrimaryNewsListItem({Key key, @required this.news})
+      : assert(news != null),
+        super(key: key);
+
+  @override
+  Widget build(BuildContext context) => Stack(
+        children: <Widget>[
+          ClipRRect(
+            borderRadius: BorderRadius.circular(_Constants.newsImageRadius),
+            child: Container(
+              color: Colors.black,
+              child: Opacity(
+                opacity: _Constants.newsImageLayerOpacity,
+                child: Image.network(
+                  news.imageUrl,
+                  //FIXME enter correct dimensions when filling a list with those items
+                  width: 300,
+                  height: 300,
+                  fit: BoxFit.fill,
+                ),
+              ),
+            ),
+          ),
+          Positioned(
+            top: 1,
```

Comment:
tak z ciekawości, czemu 1, a nie 0? :P

Reply:
w sumie to tak sobie jebnąłem, bo miało być różne od null. Może być 0, bo widzę, że częściej tak się to robi

Reply:
kumam, generalnie na początku jak spojrzałem to pomyślałem, że wartości są tu od 0 do 1 i o to w tym chodzi, a to po prostu odległość od danej krawędzi :D